zaas-service/src/main/java/org/zowe/apiml/zaas/security/config/NewSecurityConfiguration.java

```diff
@@ -522,8 +522,6 @@ public SecurityFilterChain certificateOrAuthEndpointsFilterChain(HttpSecurity ht
                     http.x509(withDefaults())
```

Comment:
the AT-TLS option is still using it, does it make sense?

Reply:
probably not as we don't want to use the default spring security, I will remove it.

Reply:
pretty sure the whole if is not needed since it's not used anywhere else in this file. Unless there's something very specific about this endpoint in AT-TLS ofc.
But we should test it out